client/ios/Podfile.lock

```diff
@@ -182,6 +182,8 @@ PODS:
     - React-cxxreact (= 0.61.5)
     - React-jsi (= 0.61.5)
   - React-jsinspector (0.61.5)
```

Comment:
we don't need to commit Podfile.lock, the packages should be installed whenever we use yarn install and then go to ios folder to instal pods.

Reply:
ok